### DISCUSSION TITLE
关于NLP和LLM的学习思路

### DISCUSSION THREAD

**Discussion by JPQuan** [datawhalechina/DOPMC#235] [问题求助]
团队您好，我是本科接触点cv领域，有机器学习的一定基础。现在想找一份算法岗的实习，因为我是海外读研，没有加入到实验室内，所以目前身边没有什么同学了解这方面的内容。我想开始着手学习NLP和LLM这方面，请问有什么建议吗。

**Comment by ningshixian** [comment]
LLM 学习资料
● 【深度学习人类语言处理2020】李宏毅
  ○ 语音识别、BERT 家族、NAG、QA, GPT3、Controllable Chatbot、Dialogue State Tracking
  ○ [YouTube 视频课程](https://www.youtube.com/playlist?list=PLJV_el3uVTsO07RpBYFsXg-bN5Lu0nhdG)、https://speech.ee.ntu.edu.tw/~tlkagk/courses_DLHLP20.html
● 【[80分鐘快速了解大型語言模型](https://www.youtube.com/watch?v=wG8-IUtqu-s)】李宏毅 
● [大语言模型综述](https://github.com/RUCAIBox/LLMSurvey)
  ○ 大语言模型迄今为止最好的学术向中文综述。
  ○ 中文版本：[LLM_Survey_Chinese_0418.pdf](https://github.com/ninehills/blog/files/11913434/LLM_Survey_Chinese_0418.pdf)
● State of GPT：GPT 联合创始人做的演示，极好的总结了 GPT 的训练和应用。
  ○ 视频：https://www.youtube.com/watch?v=bZQun8Y4L2A
  ○ PPT：https://karpathy.ai/stateofgpt.pdf
